python/astro_metadata_translator/translators/megaprime.py

```diff
@@ -22,19 +22,6 @@
 from .fits import FitsTranslator
 from .helpers import tracking_from_degree_headers, altaz_from_degree_headers
 
-filters = {'u.MP9301': 'u',
```

Comment:
Why is this change part of this PR?

Reply:
Because I noticed it was a problem and didn't feel it was worthy of an entirely new Jira ticket to remove a variable. We are allowed to do minor clean up commits (so long as they are separate commits) with other tickets.